src/components/Picker/index.js

```diff
@@ -58,13 +77,14 @@ class Picker extends PureComponent {
                         onClose={() => this.setState({isOpen: false})}
                         disabled={this.props.isDisabled}
                         focused={this.state.isOpen}
-                        hasError={this.props.hasError}
+                        errorText={this.props.errorText}
+                        value={this.props.value}
                         // eslint-disable-next-line react/jsx-props-no-spreading
                         {...pickerProps}
                     />
                 </View>
                 {!_.isEmpty(this.props.errorText) && (
-                    <InlineErrorText>
+                    <InlineErrorText style={[styles.ml3]}>
```

Comment:
This should also be applied from right edge as well. 
```suggestion
                    <InlineErrorText style={[styles.mh3]}>
```